coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
will look into possibly using .gitignore to eliminate these extra lines and file info